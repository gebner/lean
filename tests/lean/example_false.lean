open expr tactic

example : false := by do
n ← mk_fresh_name,
apply (local_const n n binder_info.default (const ``false []))
