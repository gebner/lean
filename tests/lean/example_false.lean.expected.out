example_false.lean:3:0: error: failed to add declaration to environment, it contains local constants
